django_datawatch/admin.py

```diff
@@ -1,15 +1,23 @@
 from django.contrib import admin
 
-from django_datawatch.models import Result, CheckExecution, ResultStatusHistory
+from django_datawatch.models import Result, CheckExecution, ResultAssignedGroup, ResultAssignedUser, ResultStatusHistory
 
 
+class ResultAssignedGroupInline(admin.TabularInline):
+    model = ResultAssignedGroup
+    extra = 0
+
+class ResultAssignedUserInline(admin.TabularInline):
+    model = ResultAssignedUser
+    extra = 0
+
 @admin.register(Result)
 class CheckAdmin(admin.ModelAdmin):
     list_display = ('slug', 'identifier', 'status')
     readonly_fields = ('created', 'modified')
     search_fields = ('slug', 'identifier', 'payload_description')
-    list_filter = ('status', 'slug', 'assigned_to_group')
-
+    list_filter = ('status', 'slug', 'assigned_groups', 'assigned_users')
```

Comment:
Not sure if it makes sense to add list filters for assigned users, those could be hundreds or thousands.